qcschema/versions.py

```diff
@@ -3,11 +3,12 @@
 from the development branch
 """
 
+import json
 import os
 
 from . import dev
 
-_data_path = os.path.dirname(__file__)
+_data_path = os.path.dirname(__file__) + os.sep + "data"
```

Comment:
Recommend using Path objects here.

Reply:
isn't qcsk trying to maintain py27 compatibility? or was that a misinterpretation of the CI